### PR TITLE
CM-274: added Payroll status

### PR DESCRIPTION
**Pull request by sniedzielski**
TICKET: https://openimis.atlassian.net/browse/CM-274

### CODE REVIEW

File: src/components/payroll/PayrollStatusPicker.js [hunk 1]
```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import { ConstantBasedPicker } from '@openimis/fe-core';
+
+import { PAYROLL_STATUS_LIST } from '../../constants';
+
+function PayrollStatusPicker(props) {
+  const {
+    required, withNull, readOnly, onChange, value, nullLabel, withLabel,
+  } = props;
```
Comment (olewandowski1) [inline src/components/payroll/PayrollStatusPicker.js:9]: This seems to be quite long. Maybe better aproach would be something like that: 
`
function PayrollStatusPicker({
required,
withNull,
readOnly,
onChange,
value,
nullLabel,
withLabel,
}) {
return (...)
}
`
Edit: I see that in provided code snippet props are still in one line, I meant to be them one below the other.

**Review by olewandowski1 — APPROVED** [review]
@sniedzielski I left one comment, but this is not a 'must have'. Let me know what do you think.

File: src/components/payroll/PayrollStatusPicker.js (hunk 1 above)
Comment (sniedzielski) [reply src/components/payroll/PayrollStatusPicker.js:9]: I changed this too in dependent PR

**Comment by sniedzielski** [comment]
@olewandowski1 Here https://github.com/openimis/openimis-fe-payroll_js/pull/7 I changed base to develop therefore those changes coming from that PR should be visible in that above PR.

**Pull request merged**